open-metadata-implementation/access-services/digital-service/digital-service-spring/README.md

```diff
@@ -0,0 +1,6 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# DigitalService Open Metadata Access Service (OMAS) with Spring
+
+The Digital Service OMAS spring  uses spring to create the server-side support for the REST services.module
```

Comment:
typo - '.'
'spring uses spring' should probably be 'spring module uses spring' ...